pkg/console/starter/starter.go

```diff
@@ -152,6 +167,15 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		resourceSyncer,
 	)
 
+	cliDownloadsController := clidownloads.NewCLIDownloadsSyncController(
```

Comment:
Lets organize these with `client`/`informer` next to each other, `client` first like https://github.com/openshift/console-operator/pull/310/files#diff-69ecbb024dab972c8729678abdf9c0faR79.

I think it will help, granted .  don't think there will be much added to this.

```golang
cliDownloadsController := clidownloads.NewCLIDownloadsSyncController(
  // operator is to set status, correct?
  operatorConfigClient.OperatorV1(),
  operatorClient,
  // client, informer pair for ConsoleCLIDownloads
  consoleClient.ConsoleV1().ConsoleCLIDownloads(),
  consoleInformers.Console().V1().ConsoleCLIDownloads(),
  // route just to get route... but should we be informed of a route change?
  routesClient.RouteV1(),
  recorder,
)
```

Reply:
I think we prob want the `routeInformer`.  If the `Host` changes on the route, then we need to update the `CLIDownloads`.

Reply:
Check that, you'll need both for `route`:

```golang
routeClient // so we can GET the downloads route & pluck out the canonical host.
routeInformer // notify us of any route changes
```